src/database/mysql/mysql_database.cc

```diff
@@ -284,6 +284,21 @@ std::string MySQLDatabase::quote(std::string value) const
     return ret;
 }
 
+std::string MySQLDatabase::quote(std::string_view value) const
```

Comment:
Wouldn't a generic method that takes a char array and length be better? this is pretty much a copy of the above?

Reply:
It should be possible to get rid of the above method complete (with a bit of work on the calls